python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +14,9 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skipIf(
```

Comment:
E   TypeError: skipIf() missing 1 required positional argument: 'reason'

You need only one skipIf:
```Python
@skip_if_exercize_not_started(exercize)
@unittest.skipIf(socket.gethostname() == 'omalleypark-1')
class TestStackLimit(unittest.TestCase):
    ...
```